### PR TITLE
Make the input boxes for secret/secret value and variable/variable value sticky

### PR DESCRIPTION
**Pull request by KetchupOnMyKetchup**
Resolves issue:
- https://github.com/github/vscode-github-actions/issues/35

Demo GIF for secret:
vscodeinputsticky

Demo GIF for variable:
vscodeinputsticky2

### CODE REVIEW

File: src/commands/secrets/addSecret.ts [hunk 1]
```diff
@@ -12,15 +12,17 @@ export function registerAddSecret(context: vscode.ExtensionContext) {
...
       });
 
       if (!name) {
         return;
       }
 
       const value = await vscode.window.showInputBox({
-        prompt: "Enter the new secret value"
+        prompt: "Enter the new secret value",
+        ignoreFocusOut: true
```
Comment (KetchupOnMyKetchup) [inline src/commands/secrets/addSecret.ts:25]: Is there a way I could/should test this with a test or do we test UI stuff like this manually only? I didn't see any tests verifying the text in the repo

Comment (KetchupOnMyKetchup) [reply src/commands/secrets/addSecret.ts:25]: From pairing call, this is not setup yet so there aren't tests to add.

**Review by cschleiden — APPROVED** [review]
LGTM! 

We should probably do the same for the `variable` inputs

**Comment by KetchupOnMyKetchup** [comment]
> LGTM!
> 
> We should probably do the same for the `variable` inputs

That's a really great callout, let me go ahead and add it to this PR since its a quick change and very similar so we have a consistent experience

**Pull request merged**